client/src/api/managment-service.ts

```diff
@@ -41,6 +41,36 @@ export interface ClusterSummary {
   }[];
 }
 
+export interface NotificationChannel {
+  id: string;
+  name: string;
+  service: string;
+  details: string;
+  updated: string;
+  added: string;
+  [key: string]: string;
+}
+
+export interface ApplicationEntry {
+  id: string;
+  name: string;
+  precision: string;
+  customPrompt: string;
+  updated: string;
+  added: string;
+  [key: string]: string;
+}
+
+export interface NodeEntry {
+  id: string;
+  name: string;
+  precision: string;
+  customPrompt: string;
+  updated: string;
+  added: string;
+  [key: string]: string;
+}
```

Comment:
What do you think about adding methods in management api which use these interfaces?